src/components/Button/Button.jsx

```diff
@@ -2,7 +2,20 @@ import React from 'react';
 import classNames from 'classnames';
 import MyLink from '~/components/MyLink';
 
-const Button = ({ as, icon, className, children, size, type, rounded, disabled, lowerCase, block, ...rest }) => {
+const Button = ({
+  as,
+  icon,
+  className,
+  children,
+  size,
+  type,
+  rounded,
+  disabled,
+  lowerCase,
+  block,
+  htmlType,
+  ...rest
+}) => {
```

Comment:
For the new pages, I suggest using the Ant Design buttons directly. We've overwritten the stylings of AntD buttons in https://tidb-community-ui.vercel.app/?path=/story/ant-design-button--primary